hongit-api/src/test/kotlin/com/devfloor/hongit/api/docs/ArticleControllerDocs.kt

```diff
@@ -65,74 +68,154 @@ internal class ArticleControllerDocs {
-                            .attributes(emptyFormat(), optional())
+                            .optional()
                             .description("게시글의 작성자 Github 계정"),
                         fieldWithPath("author.blog").type(JsonFieldType.STRING)
-                            .attributes(emptyFormat(), optional())
+                            .optional()
                             .description("게시글의 작성자 블로그 주소"),
                         fieldWithPath("author.description").type(JsonFieldType.STRING)
-                            .attributes(emptyFormat(), optional())
+                            .optional()
                             .description("게시글의 작성자 소개"),
                         fieldWithPath("content").type(JsonFieldType.STRING)
-                            .attributes(emptyFormat(), required())
                             .description("게시글 내용"),
                         fieldWithPath("hashtags[]").type(JsonFieldType.ARRAY)
-                            .attributes(emptyFormat(), optional())
+                            .optional()
                             .description("게시글 해시태그"),
                         fieldWithPath("favoriteCount").type(JsonFieldType.NUMBER)
-                            .attributes(emptyFormat(), required())
                             .description("게시글 좋아요 갯수"),
                         fieldWithPath("wonderCount").type(JsonFieldType.NUMBER)
-                            .attributes(emptyFormat(), required())
                             .description("게시글 궁금해요 갯수"),
                         fieldWithPath("clipCount").type(JsonFieldType.NUMBER)
-                            .attributes(emptyFormat(), required())
                             .description("게시글 스크랩 갯수"),
                         fieldWithPath("createdAt").type(JsonFieldType.STRING)
-                            .attributes(dateTimeFormat(), required())
+                            .dateTimeFormat()
                             .description("게시글 생성 시간"),
                         fieldWithPath("modifiedAt").type(JsonFieldType.STRING)
-                            .attributes(dateTimeFormat(), required())
+                            .dateTimeFormat()
+                            .description("게시글 수정 시간")
+                    )
+                )
+            )
+    }
+
+    //@Test
+    internal fun `showAllByBoardId - 게시글 목록조회(게시판) 문서화`() {
+        // given
+        given(
+            articleService.showAllByBoardId(
+                anyLong(),
+                anyInt(),
+                anyInt(),
+                ArticleSortType.FAVORITE,
+                anyList()
+            )
+        ).willReturn(listOf(ARTICLE_FEED_RESPONSE_1))
+
+        val params = LinkedMultiValueMap<String, String>()
+        params.add("boardId", BOARD_1.id.toString())
+        params.add("size", "1")
+        // when - then
+        mockMvc
+            .perform(
+                RestDocumentationRequestBuilders.get("$ARTICLE_API_URI")
```

Comment:
이렇게 수정해도 무방하다고 나오네요 !
```suggestion
                RestDocumentationRequestBuilders.get(ARTICLE_API_URI)
```

Reply:
수정했습니다!